remix-tests/examples/simple_storage_test.sol

```diff
@@ -1,5 +1,4 @@
 pragma solidity ^0.4.7;
-import "./tests.sol";
 import "./simple_storage.sol";
```

Comment:
how will the Assert work here without this line?

Reply:
If a `.sol` file has a `_test` name suffix and if it does not contain `import "./tests.sol";` or `import "./remix_tests.sol";` `remix-tests` module will inject `import "./remix_tests.sol";` statement into it.